src/Routing.tsx

```diff
@@ -14,6 +16,8 @@ const Routing: React.FC = (): JSX.Element => (
     <Routes>
       <Route path='/login' element={<LoginPage />} />
       <Route path='/*' element={<MainPage />} />
+      <Route path='/mbtiTest' element={<MbtiTestPage />} />
+      <Route path='/mbtiTestResult' element={<MbtiTestResult />} />
```

Comment:
```suggestion
      <Route path='/mbtitest' element={<MbtiTestPage />} />
      <Route path='/mbtitestresult' element={<MbtiTestResult />} />
```
https://velog.io/@pjh612/REST-API-URI-%EA%B7%9C%EC%B9%99

restful api 규칙중에 소문자로만 하는 규칙이있는데 소문자로 바꿔보시는건 어떨까요?

Reply:
아하 확인했습니다!